packages/ui/src/BaseInput.tsx

```diff
@@ -0,0 +1,124 @@
+import classNames from 'classnames'
+import React, { forwardRef, FunctionComponent, HTMLProps } from 'react'
+import styled from 'styled-components'
+
+import { Fonts } from './styles/fonts'
+
+/**
+ * BaseInput props.
+ */
+export interface Props extends HTMLProps<HTMLInputElement> {
+  /**
+   * Status error or success will paint input border and info text to red or green respectively
+   */
+  status?: 'success' | 'error' | undefined
+  /**
+   * Input info that will appear below the input field
+   */
+  info?: string
+}
+
+export const BaseInput: FunctionComponent<Props> = forwardRef(
+  (
+    {
+      type = 'text',
+      className = '',
+      disabled = false,
+      info = '',
+      ...props
+    }: Props,
+    ref,
+  ) => {
+    return (
+      <Wrapper>
+        <InnerWrapper
+          className={classNames(
+            className && className,
+            disabled && 'disabled',
+            props.status && props.status,
+          )}
+        >
+          <input
+            ref={ref}
+            type={type}
+            {...props}
+            disabled={disabled}
+            value={props.value}
+          />
+        </InnerWrapper>
+
+        {info && <Info status={props.status}>{info}</Info>}
+      </Wrapper>
+    )
+  },
+)
+
+const Wrapper = styled.div`
+  width: 100%;
+`
+
+const InnerWrapper = styled.div`
+  input {
+    width: calc(100% - 1.125rem * 2);
+    height: 1.875rem;
+    background: ${({ theme }) => theme.inputBackground};
+    border: 1px solid ${({ theme }) => theme.inputBorder};
+    border-radius: 5px;
+    outline: none;
+    font-size: 1rem;
+    line-height: 19.36px;
+    font-weight: 600;
+    padding: 0 1.125rem;
+    font-family: ${Fonts.Inter};
+    color: ${({ theme }) => theme.inputTextColor};
+
+    &:hover {
+      background-color: ${({ theme }) => theme.inputHoverBackground};
+    }
+
+    &:active {
+      background-color: ${({ theme }) => theme.inputBackground};
+    }
+
+    &:active,
+    &:focus {
+      border-width: 2px;
```

Comment:
we shouldn't increase border-width when changing state (focus) (because we are changing layout from outside or inside).
We should have box-shadow that will act as additional border. Or remove border totally, and use box-shadow instead. 
Maybe even outline would do the job, but for me personally I think box-shadow is better.